prism/prism.py

```diff
@@ -34,20 +35,22 @@ class PrismXBlock(XBlock):
         ],
         scope=Scope.settings
     )
+    
+    theme = String(
+        help="Theme selector to select prism theme",
+        default="dark",
+        values=[
+            {'display_name': 'light', 'value': 'light'},
+            {'display_name': 'dark', 'value': 'light'}
+        ],
```

Comment:
Same goes for the language choices.

```suggestion
        values=THEME_CHOICES,
```